prediction_market_agent/agents/known_outcome_agent/deploy.py

```diff
@@ -37,6 +38,12 @@ def pick_markets(self, markets: list[AgentMarket]) -> list[AgentMarket]:
             # been correctly bet on, and therefore the value of betting on them
             # is low.
             if not market_is_saturated(market=market):
+                if not has_question_event_happened_in_the_past(
+                    model=self.model, question=market.question
+                ):
+                    # event happened
```

Comment:
Wait, event happened but we are skipping it?

Reply:
That's misleading, adjusted thanks